.bcr/source.template.json

```diff
@@ -0,0 +1,5 @@
+{
+  "integrity": "",
+  "strip_prefix": "{REPO}-{VERSION}",
+  "url": "https://github.com/{OWNER}/{REPO}/releases/download/{TAG}/capnproto-cpp-{VERSION}.zip"
```

Comment:
To properly use this library downstream, the repo must use GitHub releases, and must archive each release at this name, as a download artifact on the release.

This ensures a stable hash even if GitHub [changes how their endpoints work](https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/).

The `on.release.yml` flow will publish the archives at this URL automatically when a GitHub release is published.